sailor/assetcentral/functional_location.py

```diff
@@ -0,0 +1,123 @@
+"""
+Retrieve Functional Location information from AssetCentral.
+
+Classes are provided for individual Functional Locations as well as groups
+of Functional Locations (FunctionalLocationSet).
+"""
+
+from sailor import _base
+from ..utils.timestamps import _string_to_timestamp_parser
+from .utils import _ac_fetch_data, AssetcentralEntity, _AssetcentralField, AssetcentralEntitySet, \
+    _ac_application_url
+from .constants import VIEW_FUNCTIONAL_LOCATIONS
+
+
+_FUNCTIONAL_LOCATION_FIELDS = [
+    _AssetcentralField('name', 'name'),
+    _AssetcentralField('internal_id', 'internalId'),
```

Comment:
at least for `Equipment` there's both `name` and `internalId` but they always return the exact same value, so we're returning only one and ignoring the other. Is it the same here, maybe?

Reply:
I guess it is the same behavior here. At least I can confirm for some samples I checked.